docs/self-managed/platform-deployment/helm-kubernetes/overview.md

```diff
@@ -11,16 +11,20 @@ There are many ways you can provision and configure a Kubernetes cluster, and th
 
 ## Kubernetes environments
 
-You can install Camunda 8 on your Kubernetes environment of choice, e.g.:
+You can install Camunda 8 on your [CNCF Certified Kubernetes](https://www.cncf.io/training/certification/software-conformance/) environment of choice, e.g.:
 
-- [Stock Kubernetes](https://kubernetes.io/docs/).
 - [Kubernetes KIND](https://github.com/kubernetes-sigs/kind), Minikube, K3s, and MicroK8s for local development.
-- [Red Hat OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift), an enterprise ready solution with a focus on security.
-- Cloud service providers like Google GKE, Azure AKS, Amazon EKS, etc.
+- [Red Hat OpenShift](./platforms/redhat-openshift.md), an enterprise ready solution with a focus on security.
+- Cloud service providers:
+  - [Amazon EKS](./platforms/amazon-eks/amazon-eks.md)
+  - [Google GKE](./platforms/google-gke.md)
+  - [Azure AKS](./platforms/microsoft-aks.md)
 
 ## Officially supported platforms
 
-With the right configuration, Camunda 8 Self-Managed can be deployed on any Kubernetes distribution (Cloud or on-premise). However, we officially test and support a [specific list of platforms](./platforms/platforms.md).
+With the right configuration, Camunda 8 Self-Managed can be deployed on any Kubernetes distribution (Cloud or on-premise). 
+- Camunda 8 self-managed supports the following [CNCF Certified Kubernetes distributions](https://www.cncf.io/training/certification/software-conformance/). Some distributions include custom patches or upstream source code changes. Camunda will not provide support for issues arising from these changes.
+- We test and validate a [limited list of platforms](./platforms/platforms.md).
```

Comment:
Does the fact that we test on a limited set of k8s distributions make a difference to the customer?

Reply:
We need to be super clear about this. What do we test on, and what do we assume it will run on? 

We have pending changes on the Supported Environments page to address clarity in exactly what will get our support. If we say we test and validate on a limited list of platforms, let's name those explicitly.

Reply:
I'm good with the info available today via https://docs.camunda.io/docs/reference/supported-environments/#deployment-options. @theburi we can resolve this conversation, but it looks like there are merge conflicts with the page referenced in this conversation.

Reply:
agree, I think it makes sense to rebase this PR and resolve conflicts before getting a fresh review.